### PR TITLE
development/components fix

### LINKED ISSUE #758: Form field and box component need a fixed width property
A form element or any element that is wrapped with a box component will need a set a width depending on the layout

Form field and box component will need a width property

### LINKED ISSUE #762: Shadows on Backbutton aren't defined in the shadows atom
@annaisaacks 
[this button](https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/Dev-Launchers-Style-Guide?node-id=387%3A1683&t=3WdHZu87GLNmJEjA-4) has a box-shadow that isn't defined  in the shadows atom,
seems very similar to [shadow400](https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/Dev-Launchers-Style-Guide?node-id=483%3A3253&t=3WdHZu87GLNmJEjA-4) though!
Should we use shadow400 on Backbutton?

### PR DESCRIPTION
**Pull request by Enjoy2Live**
- Update staging image to 1f4c8c0-202212151808
- Upgrade staging image to devlaunchers/platform-website:c67e80f-202212122347
- feat(recruitment): added the new styles of the application form
- Upgrade staging image to devlaunchers/platform-website:1f4c8c0-202212151808
- added proper shadow to button Closes #762
- added width and maxWidth props to FormField & Box fixes #758
- fixed validation for name and portfolio link
- added text align prop
- fixed required state wrapping to a new line
- fix theme issues in the old pages
- Upgrade staging image to devlaunchers/platform-website:2542c66-202212181154
- added extraInfo field
- fixed the checkbox :checked state
- fixed error not showing if the field isn't required
- remove exessive padding if the input is invalid
- Upgrade staging image to devlaunchers/platform-website:a9678f2-202212181311
- Roman/hamburger-button: Fixing hamburger icon styles
- Upgrade staging image to devlaunchers/platform-website:04e2acd-202212182030
- navBar width should be auto instead of 100%. There is a horizontal scoll
- navBar logo links to homepage
- added logout and links to navbar
- Added helper component to focus on form errors
- added scroll to error functionality

### CODE REVIEW

File: packages/UI/src/utils/formErrorScroller.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import type { FormikContextType } from 'formik';
+import { connect } from 'formik';
+
+type FormErrorScrollerProps = {
+  align?: ScrollIntoViewOptions['block'];
+  focusAfterScroll?: boolean;
+} & { formik?: FormikContextType<Record<string, unknown>> };
+
+const FormErrorScroller = ({
```
Comment (Enjoy2Live) [inline packages/UI/src/utils/formErrorScroller.tsx:9]: @alicefate I made this component to automatically scroll to error input fields, 
usage: 
```jsx
import FormErrorScroller from '@devlaunchers/components/src/utils/formErrorScroller';
.
.
.
<FormErrorScroller focusAfterScroll /> {/* make sure to use this within formik rendering method */}
```
here's how we use it at recruitment app https://github.com/dev-launchers/dev-launchers-platform/pull/792/files#diff-8e96e73396d9633f0955f64672fe64175f662e854143af75c0daf2f1f54f36aa

**Review by Enjoy2Live — COMMENTED** [review]
@alicefate I made some changes to help fix most of #775 issues,
"BackButton don't have onClick" should be done soon there's a PR by @Nouri-Anouar #786

**Comment by dbradham** [comment]
@Enjoy2Live We are still blocked from deploying until this is merged. Can I merge this now? CC: @pyxld-kris

**Comment by dbradham** [comment]
Adding @Nouri-Anouar and @mfvil as reviewers since they are more familiar with the process of code review and the area of the codebase being reviewed than Iris.

**Comment by Enjoy2Live** [comment]
> @Enjoy2Live We are still blocked from deploying until this is merged. Can I merge this now? CC: @pyxld-kris
@dbradham 
is @alicefate still contributing? This PR address the issues they faced in #775 and this PR is supposed to fix them.

if not you can merge and please read my previous comments on how to use the `FormErrorScroller` component.
so that you can use it on the idea form

**Comment by dbradham** [comment]
> > @Enjoy2Live We are still blocked from deploying until this is merged. Can I merge this now? CC: @pyxld-kris
> 
> @dbradham
> 
> is @alicefate still contributing? This PR address the issues they faced in #775 and this PR is supposed to fix them. if not you can merge and please read my previous comments on how to use the `FormErrorScroller` component. so that you can use it on the idea form

Thanks! I'm going to merge now but would like to discuss this particular change on a call so I can better understand how we can work together to solve these issues. I'll be in tomorrows team lead call but if that doesn't work it may be better to schedule a one-off with just us. (I could also make co-working next week)

We will make the change per your recommendation but will release the current idea form as is because the form scrolling is not included in the product specs for that particular form and still represents a big enough change as is. I'll work with Julie to prioritize follow up work here. Thanks again for catching this!

**Review by dbradham — APPROVED** [review]
Per our discussion in the feed here, I am merging this to unblock deployments for ideaspace which includes work from multiple sprints.

**Pull request merged**